tests/features/ui/ui_data_transfer.feature

```diff
@@ -0,0 +1,81 @@
+@ui
+Feature: Data Transfer UI
+
+    Scenario Outline: Data Transfer page
+        Given user <user> is logged in
+        When user opens the Data Transfer page
+        Then Data Transfer is shown
+
+        Examples:
+            | user           |
+            | researcher     |
+            | technicaladmin |
+
+
+    Scenario Outline: User clicks on the iCommands docs page
+        Given user <user> is logged in
+        When user opens the Data Transfer page
+        Then user clicks on the iCommands docs page
+
+        Examples:
+            | user           |
+            | researcher     |
+            | technicaladmin |
+
+
+    Scenario Outline: User copies iCommands configuration
+        Given user <user> is logged in
+        When user opens the Data Transfer page
+        Then user clicks on iCommands copy button
```

Comment:
I'd suggest splitting this kind of line into 2, one When for when the user clicks the button, and a Then for the assertion that the contents were copied to the clipboard.